packages/amplify_storage_s3/lib/amplify_storage_s3.dart

```diff
@@ -45,28 +45,28 @@ class AmplifyStorageS3 extends StoragePluginInterface {
   }
 
   @override
-  Future<UploadFileResult> uploadFile({@required UploadFileRequest request}) {
```

Comment:
With `@required` removed, I think `import 'package:flutter/foundation.dart';` is now unused.

Reply:
thanks removed.